jest.config.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
     "./tests/e2e/ans/publishANSContracts.ts",
     "./confidential-assets/*",
   ],
-  testPathIgnorePatterns: ["dist/*", "examples/*"],
+  testPathIgnorePatterns: ["dist/*", "examples/*", "confidential-assets/*"],
```

Comment:
I dont want confidential-assets to be part of the ts-sdk tests, it should have its own test suites

Reply:
yes, this is what this PR accomplishes?  it does have its own test suite.

Reply:
this is testPath**Ignore**Patterns